config/jobs/kubernetes/sig-autoscaling/sig-autoscaling-presubmits.yaml

```diff
@@ -1,6 +1,6 @@
 presubmits:
   kubernetes/kubernetes:
-  - name: pull-kubernetes-e2e-gci-gce-autoscaling
+  - name: pull-autoscaling-e2e-gci-gce
```

Comment:
the pattern is `{pull,ci}-$repo`, so these should still be `pull-kubernetes-...`

but the presubmits in the autoscaler repo below should be `pull-autoscaler`

thanks!